src/main/java/cn/edu/tsinghua/tsfile/file/metadata/TimeSeriesChunkMetaData.java

```diff
@@ -53,6 +53,18 @@
     private TInTimeSeriesChunkMetaData tInTimeSeriesChunkMetaData;
     private VInTimeSeriesChunkMetaData vInTimeSeriesChunkMetaData;
 
+    /**
+     * The maximum time of the tombstones that take effect on this chunk. Only data with larger timestamps than this
+     * should be exposed to user.
+     */
+    private long maxTombstoneTime;
+
+    /**
+     * The time when the RowGroup of this chunk is closed. This will not be written out and will only be set when read together
+     * with its RowGroup during querying.
+     */
+    private long writtenTime;

```

Comment:
What's the difference of variable 'writtenTime' between class TimeSeriesChunkMetaData and RowGroupMetaData? Maybe I haven't read the code of IoTDB... I can not understand the code currently.